dev/io.openliberty.jakarta.cdi.3.0/bnd.bnd

```diff
@@ -40,9 +40,11 @@ Import-Package:  jakarta.el; version="[4.0.0,5.0.0)", \
  jakarta.enterprise.inject.spi.configurator; version=3.0, \
  jakarta.enterprise.util; version=3.0, \
  jakarta.inject; version=2.0, \
- jakarta.interceptor;version=2.0
+ jakarta.interceptor;version=2.0, \
+ com.ibm.ws.cdi.liberty
```

Comment:
This import is problematic.

There are cases where another feature might want to include the CDI API bundle, (or the CDI API feature) so that it can work with or without CDI present. I think this import will prevent the API bundle from resolving if the rest of CDI is not present.

I think the easiest way to do this would be to:
 - Add `CDIProvider` to the list of services provided by the `CDIRuntimeImpl` component
 - Remove all references to `CDIRuntimeImpl` and `CDIService` from `CDIProviderService`, just look up `CDIProvider` as a service instead
 - Move `CDIProviderService` into the API bundle.

If we didn't want to add non-API things to the API bundle, you could
 - Put the services file and `CDIProviderService` into a fragment which attaches to the API bundle
 - This would still require making the `CDIRuntimeImpl` component a `CDIProvider` service